repo_resource/test_in.py

```diff
@@ -4,22 +4,22 @@
 # Copyright 2022 (c) BayLibre, SAS
 # Author: Mattijs Korpershoek <mkorpershoek@baylibre.com>
 
+import os
 import json
 from io import StringIO
 import shutil
 import unittest
 from pathlib import Path
-
-import repo
+import xml.etree.ElementTree as ET
 
 from . import check
 from . import common
 from . import in_
 
 
 class TestIn(unittest.TestCase):
-
     def setUp(self):
+        os.chdir('/root')
```

Comment:
why ?

Reply:
Most methods start moving to the working directory and move back to the previous folder, just like cd /dir; cd -
The issue with this is we end up in the working directory forever if a method calls another method eg
cd /dir; (cd /dir; cd -); cd -, we stay in /dir...
I skipped this issue calling methods directly and removed this addition.
It'd be good to change this to some simple cd /dir; cd $HOME which would make things easier in my opinion